moto/sns/models.py

```
@@ -489,14 +489,14 @@ def delete_topic(self, arn: str) -> None:
             parsed_arn = parse_arn(arn)
             sns_backends[parsed_arn.account][parsed_arn.region].topics.pop(arn, None)
         except KeyError:
-            raise SNSNotFoundError(f"Topic with arn {arn} not found")
+            pass
```

Should the error be suppressed in responses instead of here?